var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -3,15 +3,14 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
-# Although zlib comes with a configure script, it does not use Autotools
-# The AutotoolsPackage causes zlib to fail to build with PGI
 import glob
 import os
 
 from spack.package import *
 
 
+# Although zlib comes with a configure script, it does not use Autotools
+# The AutotoolsPackage causes zlib to fail to build with PGI
```

Comment:
IMO this should be reverted (it doesn't look like there were any changes other than moving a comment)